test/lint/lint-python-dead-code.sh

```diff
@@ -15,5 +15,5 @@ fi
 
 vulture \
     --min-confidence 60 \
-    --ignore-names "argtypes,connection_lost,connection_made,converter,data_received,daemon,errcheck,is_compressed,is_valid,verify_ecdsa,msg_generic,on_*,optionxform,restype,profile_with_perf" \
-    $(git ls-files -- "*.py" ":(exclude)contrib/" ":(exclude)test/functional/data/invalid_txs.py")
+    $(git rev-parse --show-toplevel) \
+    $(dirname "${BASH_SOURCE[0]}")/lint-python-dead-code-whitelist
```

Comment:
Won't this `vulture` on the `lint-python-dead-code-whitelist` file? The intention was to use `lint-python-dead-code-whitelist` to build `--exclude`? :-)

Reply:
according to the vulture documentation this is the correct invocation

Reply:
TIL! Thanks!